runtime/queries/nix/indents.toml

```diff
@@ -0,0 +1,9 @@
+indent = [
+ "if",
+ "let",
+ "function",
+ "attrset",
+ "list",
+ "indented_string",
+ "parenthesized"
+]
```

Comment:
You likely want to define some of the brackets as outdents, we don't automatically outdent when typing those yet, but it's used in certain calculations (like if you calculated the indent of every existing line)

Reply:
I actually removed the outdents because it seemed to screw up the indents (by not indenting enough spaces)